azurerm/internal/services/cosmos/cosmosdb_account_data_source_test.go

```diff
@@ -54,6 +54,8 @@ func (CosmosDBAccountDataSourceResource) basic(data acceptance.TestData) string
 data "azurerm_cosmosdb_account" "test" {
   name                = azurerm_cosmosdb_account.test.name
   resource_group_name = azurerm_resource_group.test.name
+
+  depends_on = [azurerm_cosmosdb_account.test]
```

Comment:
this is.. odd - why is this required?

Reply:
I have upgraded my local Terraform to 0.14. And the data source acctest runs fail for lack of the declare of the dependency. But I think it should work in TC. I'll remove it.